lib/authlogic/session/cookies.rb

```diff
@@ -35,6 +35,19 @@ def cookie_key(value = nil)
 
         # If sessions should be remembered by default or not.
         #
+        # If set to false authlogic sets session coockie that browsers normally
+        # delete when the user closes the browser and it logs out him
+        # Note: The user can configure the browser do not remove session cookies,
+        # in this case he will never be auto log out. There is this option:
+        # - 'Continue where you left off' - in Chrome
+        # - 'All windows from last session' - in Safari
+        # - 'Show my windows and tabs from last time' - in FireFox
+        #
+        # If set to true authlogic sets coockie with expiration
+        #
+        # Note: if you use Rails then log out behaviour depends on Rails session
+        # configuration too, as authlogic duplicates his session in Rails session
```

Comment:
I want to mention in some way that if you set for example `remember_me: true; remember_me_for: 1.day` it does not guarantee auto-log-out in 1 day in Rails and you have check Rails session configuration. For example, if Rails session_store can have `expire_after: 2.week` you will get different behaviour